### ISSUE TITLE
nodetool getssstbales is broken

### ISSUE THREAD

**Issue by slivne** [scylladb/scylla#4619]
version 3.0.7

```
 nodetool getsstables ks tb2 5000
error: getSSTablesForKey(java.lang.String, boolean)
-- StackTrace --
java.lang.NoSuchMethodException: getSSTablesForKey(java.lang.String, boolean)
	at com.sun.jmx.mbeanserver.PerInterface.noSuchMethod(PerInterface.java:169)
	at com.sun.jmx.mbeanserver.PerInterface.invoke(PerInterface.java:135)
	at com.sun.jmx.mbeanserver.MBeanSupport.invoke(MBeanSupport.java:252)
	at com.sun.jmx.interceptor.DefaultMBeanServerInterceptor.invoke(DefaultMBeanServerInterceptor.java:819)
	at com.sun.jmx.mbeanserver.JmxMBeanServer.invoke(JmxMBeanServer.java:801)
	at com.scylladb.jmx.utils.APIMBeanServer.invoke(APIMBeanServer.java:167)
	at javax.management.remote.rmi.RMIConnectionImpl.doOperation(RMIConnectionImpl.java:1468)
	at javax.management.remote.rmi.RMIConnectionImpl.access$300(RMIConnectionImpl.java:76)
	at javax.management.remote.rmi.RMIConnectionImpl$PrivilegedOperation.run(RMIConnectionImpl.java:1309)
	at javax.management.remote.rmi.RMIConnectionImpl.doPrivilegedOperation(RMIConnectionImpl.java:1401)
	at javax.management.remote.rmi.RMIConnectionImpl.invoke(RMIConnectionImpl.java:829)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at sun.rmi.server.UnicastServerRef.dispatch(UnicastServerRef.java:357)
	at sun.rmi.transport.Transport$1.run(Transport.java:200)
	at sun.rmi.transport.Transport$1.run(Transport.java:197)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.rmi.transport.Transport.serviceCall(Transport.java:196)
	at sun.rmi.transport.tcp.TCPTransport.handleMessages(TCPTransport.java:573)
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run0(TCPTransport.java:834)
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.lambda$run$0(TCPTransport.java:688)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.rmi.transport.tcp.TCPTransport$ConnectionHandler.run(TCPTransport.java:687)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

```

Yet the operation via rest apii is working

```
curl -X GET "http://localhost:10000/column_family/sstables/by_key/ks%3Atb2?key=5000"
["/var/lib/scylla/data/ks/tb2-41b29680933a11e9a680000000000001/ks-tb2-ka-18-Data.db"][root@8c6620b01a8f /]# 
```

**Comment by slivne** [comment]
@amnonh lets add the dtest that we are missing to make ssure it stays working :)

**Comment by slivne** [comment]
@amnonh - is this fixed (without the hex format)

**Comment by amnonh** [comment]
@slivne Two PR are waiting to be merged

**Comment by tzach** [comment]
@amnonh ping

**Comment by amnonh** [comment]
I've sent everything, it's pending merge

**Comment by slivne** [comment]
the fix for nodetool getsstables was merged scylladb/scylla-jmx@ff0723abc6c2dd23313f93cd5ec396ffd731d743

We just merged the dtest to next 

@amnonh - please open an issue (enhancement) - to support an additional key format that cassandra added 

closing this issue
